### PR TITLE
[NFC] Fix spelling mistake

### PR DESCRIPTION
**Pull request by ldrumm**
s/PI_MEM_ALLOC_WRTITE_COMBINED/PI_MEM_ALLOC_WRITE_COMBINED/g

### CODE REVIEW

**Comment by ldrumm** [comment]
Incidentally, we may want to enact a policy on line endings. changing between CRLF and LF every time someone makes changes from windows is somewhat distracting

**Comment by aelovikov-intel** [comment]
> Incidentally, we may want to enact a policy on line endings. changing between CRLF and LF every time someone makes changes from windows is somewhat distracting

Ouch... Can you split this in two PRs please?

File: sycl/plugins/unified_runtime/pi2ur.hpp [hunk 1]
```diff
@@ -1,5193 +1,5193 @@
-//===---------------- pi2ur.hpp - PI API to UR API  ------------------------==//
```
Comment (cperkinsintel) [inline sycl/plugins/unified_runtime/pi2ur.hpp:1]: why is this 5000 line file being completely replaced for a spelling change PR?

**Comment by ldrumm** [comment]
On Thu Jan 18, 2024 at 11:12 PM GMT, Chris Perkins wrote:
> why is this 5000 line file being completely replaced for a spelling
> change PR?

As I mentioned above, this is a previous \r\n line-ending issue that's not
displayed by github well. I'll submit a patch to clean up line endings in a
separate PR, then rebase this patch.

**Comment by ldrumm** [comment]
PI is dead. Closing

**Pull request closed without merging**